.changeset/moody-kangaroos-melt.md

```diff
@@ -0,0 +1,6 @@
+---
+'@web3-ui/hooks': minor
```

Comment:
Shouldn't this be a patch rather than a minor bump since we are only fixing something? Would love to hear about your reasoning behind this if you think otherwise since I'm still pretty new to semantic versioning

Reply:
Nope! `core` is `minor` because we're fixing a bug. `hooks` is `minor` because the only thing we're actually doing in that package is exporting something new in the API. 

It helps to think about packages in a monorepo like they're actually in separate repos. If `@web3-ui/core` wanted to use `ERC20ABI` and it wasn't in the same repo, you'd have to add it as a new feature to `@web3-ui/hooks` first.

Reply:
Also even if it was fixing a bug in `/hooks` too, you take the highest version of the changes.

breaking change = `major`
backwards compatible functionality = `minor`
bug fix = `patch`

breaking change + bug fix = `major`
backwards compatible functionality + bug fix = `minor`

and so on!

Reply:
@with-heart thanks for writing that up. TIL!